tools/cli/commands.js

```diff
@@ -522,7 +522,7 @@ export const AVAILABLE_SKELETONS = [
 
 main.registerCommand({
   name: 'create',
-  maxArgs: 1,
+  maxArgs: 2,
```

Comment:
Are you sure this number needs to be increased?

Reply:
https://github.com/meteor/meteor/blob/4f16220fa648660ed24fde36f99942e89d7f6c8a/tools/cli/main.js#L163
yeah, I miss understood what I read :( We can have as many options as we want but obligatory args is just one (the path)